src/BCrypt/BCrypt.cs

```diff
@@ -234,6 +282,37 @@ public static partial class BCrypt
             int cbInput,
             BCryptHashDataFlags dwFlags = BCryptHashDataFlags.None);
 
+        /// <summary>
+        /// The BCryptProcessMultiOperations function processes a sequence of operations on a multi-object state.
+        /// </summary>
+        /// <param name="hHash">
+        /// The handle of the hash or MAC object to use to perform the operation. This handle is obtained by calling the <see cref="BCryptCreateMultiHash(SafeAlgorithmHandle, out SafeHashHandle, int, byte[], int, byte[], int, BCryptCreateHashFlags)"/> function.
+        /// </param>
+        /// <param name="operationType">
+        /// Currently the only defined value is <see cref="MultiOperationType.BCRYPT_OPERATION_TYPE_HASH"/>.
+        /// </param>
+        /// <param name="pOperations">
+        /// A pointer to an array of operation command structures  <see cref="BCRYPT_MULTI_HASH_OPERATION"/>.
+        /// </param>
+        /// <param name="cbOperations">
+        /// The size, in bytes, of the pOperations array.
+        /// </param>
+        /// <param name="dwFlags">
+        /// Specify a value of zero (0).
+        /// </param>
+        /// <returns>Returns a status code that indicates the success or failure of the function.</returns>
+        /// <remarks>
+        /// Each element of the pOperations array contains instructions for a particular computation to be performed on a single element of the multi-object state. The functional behavior of BCryptProcessMultiOperations is equivalent to performing, for each element in the multi-object state, the computations specified in the operations array for that element, one at a time, in order.
+        /// The relative order of two operations that operate on different elements of the array is not guaranteed.If an output buffer overlaps an input or output buffer the result is not deterministic.
+        /// </remarks>
+        [DllImport(nameof(BCrypt), SetLastError = true)]
+        public static extern NTSTATUS BCryptProcessMultiOperations(
+            SafeHashHandle hHash,
+            MultiOperationType operationType,
+            BCRYPT_MULTI_HASH_OPERATION[] pOperations,
```

Comment:
This should be `[Friendly(In|Array)] BCRYPT_MULTI_HASH_OPERATION*` instead of an array, since the underlying native type is just a pointer, and pointers give more flexibility to the C# caller as well. The `[Friendly]` attribute will cause overloads of the method to automatically generate, including the array one you have here. 
```suggestion
            [Friendly(FriendlyFlags.In | FriendlyFlags.Array)] BCRYPT_MULTI_HASH_OPERATION* pOperations,
```

Reply:
I've taken care of this.